scripts/set_requester_pays.sh

```diff
@@ -32,13 +28,15 @@ echo "Gatorcounting for 10 seconds while iam change goes into effect"
 echo ""
 echo "NOTE: if you get an error message saying:"
 echo "    AccessDeniedException: 403 ${USER_EMAIL} does not have storage.buckets.update access to the Google Cloud Storage bucket."
-echo "THEN gatorcount 10 more seconds and run this again."
+echo "THEN don't worry, just wait until it shows 'Enabling requester pays."
 echo ""
 sleep 10
 
 for BUCKET in $BUCKETS; do
-  # set requester pays
-  gsutil requesterpays set on ${BUCKET} || exit 1
+  while ! gsutil requesterpays set on ${BUCKET}
```

Comment:
what happens if we get a non-403 error? can we add a timeout in case something else is wrong, so this doesn't go on forever if there's something else wrong besides a permissions issue?

Reply:
your changes resolve the timeout issue, but what about a non-403 error?

Reply:
Marianie confirmed that the error will be printed, so if it's not a 403, the user will know.